nexus/reconfigurator/planning/src/blueprint_builder.rs

```diff
@@ -142,6 +195,9 @@ impl<'a> BlueprintBuilder<'a> {
             .sleds
             .keys()
             .map(|sled_id| {
+                // XXX: do we need to handle, or check for, initially expunged
```

Comment:
I would say no.  The only thing we could do here is mark zones expunged if we find them on sleds that are expunged.  But that's more of the planner's job.  The idea behind the initial blueprint is that if you executed it, it wouldn't do anything.  (That's why this is in BlueprintBuilder, which is not supposed to be that smart, and not Planner.)